gemfiles/libraries.gemfile

```diff
@@ -63,5 +43,7 @@ else
   gem 'pg'
 end
 
-gemspec :path => File.expand_path(File.dirname(__FILE__) + '/../')
+eval(File.read(File.join(File.dirname(__FILE__), 'test_gems.gemfile')))
+
+gemspec :path => File.dirname(File.dirname(__FILE__))
```

Comment:
Question, why did this one have the '/../' removed when the other files still have the '/../'?

Reply:
I had to look twice, they are actually the same.
I should have changed all of them, they both go up the hierarchy one level, the second version is more conventional and doesn't require `expand_path` (because there is no need to translate '/../').